src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -420,19 +420,15 @@ def return_response(self):
     def make_bundles(self, entry):
         return [{"bundleUuid": b["uuid"], "bundleVersion": b["version"]} for b in entry["bundles"]]
 
-    def make_processes(self, entry):
-        processes = []
-        for process in entry["contents"]["processes"]:
+    def make_protocols(self, entry):
+        protocols = []
+        for protocol in entry["contents"]["protocols"]:
             translated_process = {
-                "processId": process["process_id"],
-                "processName": process.get("process_name", None),
-                "libraryConstructionApproach": process.get("library_construction_approach", None),
-                "instrumentManufacturerModel": process.get("instrument_manufacturer_model", None),
-                "protocolId": process.get("protocol_id", None),
-                "protocol": process.get("protocol_name", None),
+                "libraryConstructionApproach": protocol.get("library_construction_approach"),
```

Comment:
```suggestion
                "libraryConstructionApproach": protocol.get("library_construction_approach", []),
```

As I mentioned earlier today, if library_construction_approach is a list (which it has to be in aggregates), a better default is the empty list.